### PR TITLE
Reduce severity classes

### PR DESCRIPTION
**Pull request by janowagner**
The support for multiple severity class ranges is removed.
There is now only one classification scheme (the one previously
called "nist"). Actually there was only one other classification scheme called
"pci-dss". It wasn't used or needed.

With applying a single default scheme code and database operations can
be simplified.

Basically all code elements are removed, including a migrator
that removes any such setting from the database.

The "report" XML element does not include the severity classification with
its range details (as element ""severity_class) anymore and the "authenticate_response"
XML does not include the "severity" element anymore.

**Checklist**:

- [ ] Tests N/A
- [x] [CHANGELOG](https://github.com/greenbone/gvmd/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

File: src/manage_pg.c [hunk 1]
```diff
@@ -1443,12 +1443,10 @@ manage_create_sql_functions ()
            "                                              text,"
            "                                              text)"
            " RETURNS boolean AS $$"
-           "              (CASE lower ($2)"
+           "              (SELECT CASE lower ($2)"
            "               WHEN 'high'"
            "               THEN $1 >= 7"
            "                    AND $1 <= 10"
-           "               WHEN 'medium'"
```
Comment (mattmundell) [inline src/manage_pg.c:1476]: Missing medium now?

File: src/manage_pg.c [hunk 2]
```diff
@@ -1443,12 +1443,10 @@ manage_create_sql_functions ()
            "                                              text,"
            "                                              text)"
            " RETURNS boolean AS $$"
-           "              (CASE lower ($2)"
+           "              (SELECT CASE lower ($2)"
```
Comment (mattmundell) [inline src/manage_pg.c:1445]:
```suggestion
           "  SELECT CASE lower ($2)"
```

File: src/manage_pg.c [hunk 3]
```diff
@@ -1443,12 +1443,10 @@ manage_create_sql_functions ()
...
            "               WHEN 'high'"
```
Comment (mattmundell) [inline src/manage_pg.c:1446]:
```suggestion
           "         WHEN 'high'"
```
etc

File: src/manage_pg.c [hunk 4]
```diff
@@ -1462,10 +1462,20 @@ manage_create_sql_functions ()
       sql ("CREATE OR REPLACE FUNCTION severity_in_level (double precision,"
            "                                              text)"
            " RETURNS boolean AS $$"
-           " SELECT severity_in_level"
-           "         ($1,"
-           "          $2,"
-           "          'dummy')"
+           "              (SELECT CASE lower ($2)"
```
Comment (mattmundell) [inline src/manage_pg.c:1445]: Indenting as before.

**Review by mattmundell — CHANGES_REQUESTED** [review]

**Comment by mattmundell** [comment]
Also, it's missing the changes to the GMP doc.

File: src/manage_pg.c (hunk 1 above)
Comment (janowagner) [reply src/manage_pg.c:1476]: Fixed, this was a mistake.

**Review by mattmundell — APPROVED** [review]

**Pull request merged**